### DISCUSSION TITLE
How can I split twice by one hit ?

### DISCUSSION THREAD

**Discussion by ivaquero** [wezterm/wezterm#2388] [Q&A] [answered]
How can I split twice by one hit to get a layout in the following screenshot ?

<img width="1435" alt="image" src="https://user-images.githubusercontent.com/46377665/184059558-f988e4ff-0601-4b57-90ab-60f5d75ba32b.png">

I try to call `wezterm.action.SplitPane` twice with the same keybinding, but found it the later one overwrite the previous. So, will WezTerm's keybinding support multi-actions ?

**Comment by wez** [comment, marked as answer]
Take a look at https://wezfurlong.org/wezterm/config/lua/gui-events/gui-startup.html
That shows how to get wezterm to start up with the arrangement that you like.

If you want to do that in response to a keypress, you can instead use https://wezfurlong.org/wezterm/config/lua/wezterm/action_callback.html and do something like:

```lua
action = wezterm.action_callback(function(win, pane)
  local new_pane = pane:mux_pane():split{}
  new_pane:split{}
end),
```

* https://wezfurlong.org/wezterm/config/lua/pane/index.html - the GUI-layer pane passed to the callback
* https://wezfurlong.org/wezterm/config/lua/MuxPane/index.html - the pane returned from the GUI pane:mux_pane() method

**Reply by ivaquero** [reply]
Thank you for the quick reply, action_callback is the thing that I need.

**Comment by wez** [comment]
In addition to action_callback, you can use [Multiple](https://wezfurlong.org/wezterm/config/lua/keyassignment/Multiple.html) to list a series of actions

**Reply by ivaquero** [reply]
I will try, thank you.
